lib/tasks/move_profile.rake

```diff
@@ -1,7 +1,7 @@
 namespace :moves do
   desc 'Set all profile ids in all the moves'
   task set_profiles: :environment do
-    moves = Move.where(profile_id: nil).includes(person: :profiles)
+    moves = Move.where(profile_id: nil)
```

Comment:
Could we have some better naming. For the file, maybe call it move_set_profile_from_person.rake.
Instead of the local variable moves, call it moves_with_nil_profile

Reply:
np, I've called this `set_profile_from_person` since it is already inside the namespace `moves`.
Anyway, after a while this tasks will be removed.